test/unit/specs/methods.spec.js

```diff
@@ -1,14 +1,40 @@
 const AppleSignin = require('ti.applesignin');
-
+var signInButton = AppleSignin.createLoginButton({
+	type: AppleSignin.BUTTON_TYPE_SIGNIN,
+	style: AppleSignin.BUTTON_STYLE_BLACK,
+	borderRadius: 10,
+});
 describe('ti.applesignin', () => {
 	describe('methods', () => {
 		describe('#authorize()', () => {
 			it('is a function', () => {
 				expect(AppleSignin.authorize).toEqual(jasmine.any(Function));
 			});
-			// TODO: verify return value is one of the RESULT_* constants!
 		});
 
-		// TODO: verify 'login' event
+		describe('#createLoginButton()', () => {
+			it('is a function', () => {
+				expect(AppleSignin.createLoginButton).toEqual(jasmine.any(Function));
+			});
+		});
+
+		describe('#getCredentialState()', () => {
+			it('is a function', () => {
+				expect(AppleSignin.getCredentialState).toEqual(jasmine.any(Function));
+			});
+		});
+	});
+});
+describe('ti.applesignin.LoginButton', () => {
```

Comment:
I think it's good practice to add 'Methods' or 'Property' instead of 'ti.applesignin.LoginButton' in describe block.